README.md

```diff
@@ -68,6 +68,21 @@ vim example.py
 pipenv run python2 example.py
 ```
 
+# Create an Accumulo client using Ruby
+
+Run the command below to create an example Ruby client:
+
+```bash
+mkdir accumulo-client/
+cd accumulo-client/
+cp /path/to/accumulo-proxy/src/main/ruby/Gemfile .
+vim Gemfile # Set correct path
+cp /path/to/accumulo-proxy/src/main/ruby/client.rb .
+gem install bundler
```

Comment:
bundler is installed as a default gem for ruby versions 2.6 and later, which is probably a good minimum version to use

Reply:
Ruby 2.5.5 is the default for my OS so I am going to leave it there